test/helper.rb

```diff
@@ -158,4 +158,11 @@ def nokogiri_fragment(str)
       str
     )
   end
+
+  def skip_if_windows(msg = nil)
+    if Utils::Platforms.really_windows?
+      msg ||= "Jekyll doesn't currently support Symlinks on Windows"
```

Comment:
This seems like a strange default. Is this only used in symlinks-related code? If not, I'd suggest something like `Jekyll does not currently support this feature on Windows.`